lib/tasks/downloads.rake

```diff
@@ -41,19 +41,25 @@ task :downloads => [:environment, :windows_downloads, :mac_downloads]
 
 task :windows_downloads => :environment do
   # find latest windows version
-  project = "msysgit"
-  win_downloads = file_downloads_from_github("msysgit/msysgit")
+  project = "git-for-windows"
+  win_downloads = file_downloads_from_github("git-for-windows/git")
   win_downloads.each do |url, date|
     name = url.split('/').last
-    if m = /^Git-(.*?)-(.*?)(\d{4})(\d{2})(\d{2})\.exe/.match(name)
-      version = m[1]
+    # Git for Windows uses the following naming system
+    # [Portable]Git-#.#.#.#[-dev-preview]-32/64-bit[.7z].exe
+    if m = /^(Portable|)Git-(\d\.\d\.\d\.\d).*-(32|64)-bit\..*exe/.match(name)
```

Comment:
This will fail if any of the first three version components becomes greater than 9.  A better option would be:
```ruby
if m = /^(Portable|)Git-(\d+\.\d+\.\d+\.\d+)-(?:.+-)*(32|64)-bit\..*exe/.match(name)
```